### PR TITLE
WD-2123 & WD-2125 - Create MongoDB product page & contact form

### PR DESCRIPTION
**Pull request by pedoch**
## Done

- Create the MongoDB product page
- Create the MongoDB contact form
- Drivebys

## QA

- Check out the [demo](https://ubuntu-com-12769.demos.haus/data/mongodb) and compare it with [the design](https://www.figma.com/file/XAmHDBtr7cjQOeDrsE4TH7/23.04-u.com%2Fdata%3A-MongoDB-%26-OpenSearch?node-id=132-2316&t=wt8d2LQ4UdWxyWGj-4) and [copy docs](https://docs.google.com/document/d/1_JuxbWsqTeQRRmSL0sCpM3alabr06WX4Cb-tt1BBW_I/edit)

## Issue / Card

- [WD-2123](https://warthogs.atlassian.net/browse/WD-2123)
- [WD-2125](https://warthogs.atlassian.net/browse/WD-2125)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-2123]: https://warthogs.atlassian.net/browse/WD-2123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WD-2125]: https://warthogs.atlassian.net/browse/WD-2125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by lyubomir-popov** [comment]
@thanks @pedoch looks really good, just a few tiny niggles listed here: https://www.figma.com/file/XAmHDBtr7cjQOeDrsE4TH7/23.04-u.com%2Fdata%3A-MongoDB-%26-OpenSearch?node-id=255%3A2195&t=Cm3qSinLsxZq3PhZ-1

please note - the talbet layout comment applies to all sections

**Comment by juanruitina** [comment]
Nice work! Is there a reason why we are not showing the tags on the blog post list?

@lyubomir-popov, I left a couple of comments in Figma related to how components play with the gray background.

**Comment by pedoch** [comment]
@lyubomir-popov @juanruitina could you take a look again? I've updated the PR.

File: templates/data/mongodb/index.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+    <hr class="u-no-margin--bottom" />
+    <p>
+      "MongoDB" is a trademark or registered trademark of MongoDB Inc. <br class="u-hide--small" />Other trademarks are property of their respective owners. Charmed MongoDB is not sponsored, endorsed, or affiliated with MongoDB, Inc.
+    </p>
+  </div>
+</section>
+
+<template style="display:none" id="mongodb-template">
+  <li class="p-list__item u-no-padding--top u-no-padding--bottom">
+    <div class="p-strip u-no-padding--top">
```
Comment (lyubomir-popov) [inline templates/data/mongodb/index.html:298]:
```suggestion
<section class="p-strip is-shallow u-no-padding--top">
```

File: templates/shared/contextual_footers/_data_mongodb_signup.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<div class="row">
+  <div class="col-3">
+    <h3 class="p-text--small-caps">Stay Informed</h3>
```
Comment (lyubomir-popov) [inline templates/shared/contextual_footers/_data_mongodb_signup.html:3]:
```suggestion
    <h3 class="p-heading--5">Stay Informed</h3>
```

File: templates/data/mongodb/index.html [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-3 col-medium-3">
+      <h2 class="p-heading--2">Resources</h2>
+    </div>
+    <div class="col-9 col-medium-3">
+      <div class="row">
+        <div class="col-3">
+          <h3 class="p-text--small-caps">Ubuntu Pro</h3>
```
Comment (lyubomir-popov) [inline templates/data/mongodb/index.html:257]:
```suggestion
          <h3 class="p-heading--5">Ubuntu Pro</h3>
```

File: templates/data/mongodb/index.html [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+            </a>
+          </p>
+        </div>
+      </div>
+      <hr class="u-no-margin--bottom" />
+      {% with item_file="shared/contextual_footers/_data_mongodb_signup.html" %} {% include item_file %} {% endwith %}
+      <hr class="u-no-margin--bottom" />
+      <div class="row">
+        <div class="col-3">
+          <h3 class="p-text--small-caps">From our blog</h3>
```
Comment (lyubomir-popov) [inline templates/data/mongodb/index.html:278]:
```suggestion
          <h3 class="p-heading--5">From our blog</h3>
```

File: templates/data/mongodb/index.html [hunk 5]
```diff
@@ -0,0 +1,319 @@
...
+          data-testid="interactive-form-link"
+        >
+          Contact us
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
```
Comment (lyubomir-popov) [inline templates/data/mongodb/index.html:248]:
```suggestion
<section>
```

**Review by lyubomir-popov — DISMISSED** [review]
A few small suggestions in the code

**Comment by juanruitina** [comment]
LGTM!

**Comment by pedoch** [comment]
@anthonydillon Sorry, I pushed a couple of changes, in case you already started your review.

File: static/sass/styles.scss [hunk 6]
```diff
@@ -1305,9 +1305,31 @@ hr.p-separator.is-shallow {
   }
 }
 
+// suru paper bottom strip styles
+.p-strip--suru-shape-paper {
```
Comment (anthonydillon) [inline static/sass/styles.scss:1309]: I don't see where this is used?

File: templates/data/mongodb/index.html [hunk 7]
```diff
@@ -0,0 +1,319 @@
...
+      </h2>
+    </div>
+    <div class="col-6 col-start-large-7 col-medium-3">
+      <p>
+        MongoDB is a document database known for its performance, resilience and security. Canonical helps you simplify MongoDB operations so you can speed up innovation.
+      </p>
+      <ul class="p-list--divided">
+        <li class="p-list__item has-bullet">World-class MongoDB consulting services</li>
+        <li class="p-list__item has-bullet">Automate operations with our Charmed MongoDB operator</li>
+        <li class="p-list__item has-bullet">Get assistance with deployment and troubleshooting</li>
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:51]:
```suggestion
        <li class="p-list__item has-bullet">Get full assistance with deployment and troubleshooting</li>
```

File: templates/data/mongodb/index.html [hunk 8]
```diff
@@ -0,0 +1,319 @@
...
+    <hr />
+    <div class="col-6 col-medium-3">
+      <h2>Consulting services</h2>
+      <p>
+        Our team of experts is here to help you get started with your database, providing guidance for new data-related projects, database builds and assistance migrating your application.
+      </p>
+      <p>
+        <a
+          href="/data/mongodb#get-in-touch"
+          class="p-button--positive js-invoke-modal"
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:75]: This button does nothing

File: templates/data/mongodb/index.html [hunk 9]
```diff
@@ -0,0 +1,319 @@
...
+    <h1 class="p-heading--2 p-heading--bold">
+      MongoDB database operations, <br class="u-hide--small u-hide--medium" />simplified
+    </h1>
+    <p>
+      Focus more on building apps, and less on maintaining the database <br class="u-hide--small" />with
+      Canonical’s Charmed MongoDB solutions.
+    </p>
+    <p>
+      <a
+        href="/data/mongodb#get-in-touch"
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:26]: This button does not work

File: templates/data/mongodb/index.html [hunk 10]
```diff
@@ -0,0 +1,319 @@
...
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6 col-medium-3">
+      <h2>Fully automated database <br class="u-hide--small u-hide--medium" />operations with Charmed <br class="u-hide--small u-hide--medium" />MongoDB</h2>
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:103]: Do you need these br's? When I remove them it renders the same for me

File: templates/data/mongodb/index.html [hunk 11]
```diff
@@ -0,0 +1,319 @@
...
+    <div
+      class="col-6 col-medium-3"
+    >
+      <p>
+        Unlock the potential of your data with an integrated, open source platform for applications and infrastructure. Canonical also offers enterprise solutions for data applications such as <a href="https://ubuntu.com/data/opensearch">OpenSearch</a>, <a href="https://ubuntu.com/data/spark">Spark</a> and <a href="https://ubuntu.com/data/kafka">Kafka</a>.
+      </p>
+      <p>
+        <a
+          href="/data/mongodb#get-in-touch"
+          class="p-button--positive js-invoke-modal"
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:238]: This button too

File: templates/data/mongodb/index.html [hunk 12]
```diff
@@ -0,0 +1,319 @@
...
+  <div class="row">
+    <hr />
+    <div class="col-6 col-medium-3">
+      <h2>Secure and scale your <br class="u-hide--small" />open-source data applications</h2>
+    </div>
+    <div
+      class="col-6 col-medium-3"
+    >
+      <p>
+        Unlock the potential of your data with an integrated, open source platform for applications and infrastructure. Canonical also offers enterprise solutions for data applications such as <a href="https://ubuntu.com/data/opensearch">OpenSearch</a>, <a href="https://ubuntu.com/data/spark">Spark</a> and <a href="https://ubuntu.com/data/kafka">Kafka</a>.
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:233]: These should be relative links not absolute

File: templates/data/mongodb/index.html [hunk 13]
```diff
@@ -0,0 +1,319 @@
...
+    articlesContainerSelector: "#mongodb-articles",
+    articleTemplateSelector: "#mongodb-template",
+    gtmEventLabel: "ubuntu.com mongodb blog",
+    limit: "3",
+    tagId: "1234",
+  })
+</script>
+
+<!-- Set default Marketo information for contact form below-->
+<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/data_mongodb" data-form-id="4960" data-lp-id="" data-return-url="/data/mongodb" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
```
Comment (anthonydillon) [inline templates/data/mongodb/index.html:316]: This is 404'ing

**Comment by pedoch** [comment]
@anthonydillon I've implemented your suggestions, could you take another look?

**Review by anthonydillon — APPROVED** [review]
LGTM thanks for the updates 👍

**Pull request merged**